logstash-core/src/main/java/org/logstash/ext/JrubyAckedWriteClientExt.java

```diff
@@ -41,67 +40,48 @@ public final class JrubyAckedWriteClientExt extends JRubyAbstractQueueWriteClien
 
     private JRubyAckedQueueExt queue;
 
-    private AtomicBoolean closed = new AtomicBoolean();
-
     @JRubyMethod(meta = true, required = 2)
     public static JrubyAckedWriteClientExt create(final ThreadContext context, final IRubyObject recv,
```

Comment:
I was very careful to make a change this Ruby method since it may _widely_ affect. However, digging a bit more shows brighter history that we had `logstash-core/lib/logstash/util/wrapped_acked_queue.rb` file before and it had [a reference for this method](https://github.com/elastic/logstash/commit/4ada6dbf4d0c2360678447a4abad80d767388d9c#diff-15bd74ca425847c432852b95f8e9e727408ddb4e019cd665fb3a52a215627508R58). It seems we migrated `wrapped_acked_queue.rb` to Java with [this change](https://github.com/elastic/logstash/pull/9111/files#diff-15bd74ca425847c432852b95f8e9e727408ddb4e019cd665fb3a52a215627508) and we no longer require this `@JRubyMethod JrubyAckedWriteClientExt create` method.

Removed and tested on local, will push changes soon and update here.

Reply:
Removed the method since unused.